haiku-apps/deeperpeople/deeperpeople-1.0.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A contacts application"
+DESCRIPTION="DeeperPeople, a contacts application for Haiku."
+APP="DeeperPeople"
+HOMEPAGE="https://github.com/HaikuArchives/$APP"
+COPYRIGHT="1991-99 Olivier Cortes"
```

Comment:
While it's true that Olivier wrote DeeperPeople, most of the code was copyrighted by Be Inc. Example: [main.cpp](https://github.com/HaikuArchives/DeeperPeople/blob/master/src/main.cpp#L10), [TTextControl.cpp](https://github.com/HaikuArchives/DeeperPeople/blob/master/src/TTextControl.cpp#L10), [PeopleWindow.cpp](https://github.com/HaikuArchives/DeeperPeople/blob/master/src/PeopleWindow.cpp#L10)

```diff
-COPYRIGHT="1991-99 Olivier Cortes"
+COPYRIGHT="1991-1999 Be Incorporated"
```